khelo/lib/ui/flow/score_board/score_board_view_model.dart

```diff
@@ -766,6 +770,28 @@ class ScoreBoardViewNotifier extends StateNotifier<ScoreBoardViewState> {
           state = state.copyWith(showInningCompleteSheet: DateTime.now());
         }
     }
+    _addPlayerStats();
```

Comment:
I don't think it's good idea to call it here as after showing any sheet, user can undo ball right, also there'll be async operation of showing sheet right, in that case if adding stat takes time, won't it results in changed stat.currentScoreList for some player?